micrometer-tracing-bridges/micrometer-tracing-bridge-otel/build.gradle

```diff
@@ -23,4 +23,5 @@ dependencies {
 	testImplementation 'org.junit.jupiter:junit-jupiter'
 	testImplementation 'org.assertj:assertj-core'
 	testImplementation 'org.mockito:mockito-core'
+	testImplementation 'ch.qos.logback:logback-classic'
```

Comment:
Is this necessary?

Reply:
because we need StaticMDCBinder to init MDC

Reply:
Where is it used?

Reply:
https://github.com/qos-ch/slf4j/blob/v_1.7.36/slf4j-api/src/main/java/org/slf4j/MDC.java#L97

Reply:
> Where is it used?

Here https://github.com/qos-ch/slf4j/blob/v_1.7.36/slf4j-api/src/main/java/org/slf4j/MDC.java#L97